helm_deploy/values-dev.yaml

```diff
@@ -55,6 +55,7 @@ generic-service:
     PRISONER_SEARCH_API_URL: "https://prisoner-search-dev.prison.service.justice.gov.uk"
     PRISON_API_URL: "https://prison-api-dev.prison.service.justice.gov.uk"
     WHEREABOUTS_API_URL: "https://whereabouts-api-dev.service.justice.gov.uk"
+    EDUCATION_AND_WORK_PLAN_API_URL: "https://learningandworkprogress-api-dev.hmpps.service.justice.gov.uk"
```

Comment:
"Education And Work Plan", "PLP" (Personal Learning Plans), and "Learning And Work Progress" are all one and the same thing, and simply reflects the change of name of our service (PLP) over time